index.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-var dotProp = require('dot-prop');
+var dotPropGet = require('dot-prop').get;
```

Comment:
Yeah, I guess there's a performance improvement to call it directly, so let us go with that. I guess the naming is ok too.